tools/release/refresh-packages.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable no-console */
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const spawn = require( 'cross-spawn' );
+const { zip, uniq, identity, groupBy } = require( 'lodash' );
+
+/**
+ * Constants
+ */
+const WORDPRESS_PACKAGES_PREFIX = '@wordpress/';
+const rootDir = __dirname + '/../..';
+const { getArgFromCLI } = require( `${ rootDir }/node_modules/@wordpress/scripts/utils` );
```

Comment:
`require( `@wordpress/scripts/utils` );` - didn't work correctly? The script runs from the project's root so, in theory, it should be all good.

Maybe, you need to pass the current working directory with `options.cwd` in the `spawn` call. See: https://nodejs.org/api/child_process.html#child_processspawncommand-args-options.

Reply:
Good point, I just removed the rootDir thing entirely as it works without it.